src/components/camera.js

```diff
@@ -43,6 +43,7 @@ module.exports.Component = registerComponent('camera', {
     if (!oldData || oldData.active !== data.active) {
       if (data.active) {
         sceneEl.setActiveCamera(camera);
+        sceneEl.emit('active-camera-set', { camera: camera });
```

Comment:
Hmm we have `camera` here. `camera.el.components` is an empty object at this point so cameraEl would actually give us less info